docs/models/mapmri.rst

```diff
@@ -1,10 +1,221 @@
 .. include:: links.rst
 
 #####################################
-Mean Apparent Propagator MRI (MAPMRI)
+Mean Apparent Propagator MRI (MAP-MRI)
 #####################################
 
+MAP-MRI Foundational Papers
+===========================
 
+**MAP-MRI Framework and Novel Diffusion Metrics**: :cite:t:`ozarslan2013` introduced MAP-MRI as a comprehensive 3D 
+q-space model that fits the diffusion signal with Hermite basis functions, 
+enabling the direct computation of the ensemble average propagator in voxel-wise 
+fashion :cite:p:`ozarslan2013`. This foundational work defined new scalar indices 
+derived from the propagator, including RTOP (Return to origin probability), 
+RTAP (Return to axis probability), RTPP (Return to plane probability), 
+propagator anisotropy (PA), and non-Gaussianity (NG) – which quantify diffusion 
+restrictions and anisotropy beyond the capabilities of DTI.
+
+The return-to-origin probability (RTOP) was defined as the probability of zero 
+displacement, sensitive to cell density and restrictions, while RTAP/RTPP 
+measure probabilities of displacements confined to an axis or plane 
+:cite:p:`ozarslan2013`. :cite:t:`ozarslan2013` also formulated propagator 
+anisotropy (PA) as a generalized anisotropy index reflecting the deviation of 
+the full propagator from an isotropic counterpart. A global non-Gaussianity (NG) 
+index was introduced to capture the total deviation of the diffusion propagator 
+from a Gaussian form, reflecting microstructural complexity not captured by a 
+single diffusion tensor :cite:p:`ozarslan2013`. :cite:t:`gu2019` demonstrated 
+the use of wild bootstrap methods to quantify uncertainty in MAP-MRI metrics 
+(RTOP, NG, PA), showing how uncertainty measures can improve group analyses and 
+aid in comparing different preprocessing pipelines.
```

Comment:
```suggestion
single diffusion tensor :cite:p:`ozarslan2013`.
```
We don't have this implemented :( so maybe we can mention this when it's possible in qsirecon